MiniApp/Classes/Display/RealMiniAppView.swift

```diff
@@ -52,12 +58,33 @@ internal class RealMiniAppView: UIView {
             wevViewBottonConstraintWithNavBar = navBar?.layoutAttachTop(to: webView)
             webViewBottomConstraintStandalone?.isActive = false
         }
-
     }
 
     required init?(coder: NSCoder) {
         return nil
     }
+  
+    override func removeFromSuperview() {
+        localServer?.stopServer()
+        localServer = nil
+        super.removeFromSuperview()
+    }
+
+    func loadWebpage(versionId: String) {
+        guard let serverUrl = localServer?.serverURL().absoluteString else {
+            return
+        }
+
+        let url = URL(string: "\(serverUrl)/\(versionId)/index.html")!
```

Comment:
Does this need to have the `versionId` in the path? This means the Mini App URL to be `https://localhost:PORT/VERSION/index.html`, but it seems this causes some Mini Apps to not work (such as the JS SDK Sample App). It's because they are referencing their assets with a preceding slash like `/styles/something.css'.

So this would work if you just remove the `versionId` from the path so that the Mini App URL is `https://localhost:PORT/index.html`.